### ISSUE TITLE
keyboard navigation does not work on tab menu

### ISSUE THREAD

**Issue by jma** [primefaces/primeng#18321]
### Describe the bug

Keyboard navigation does not work for the tab menu.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/9bfecb9g?file=src%2Fapp%2Ftabs-tabmenu-demo.ts

### Environment

Primeng: 19.1.2

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

Please press <space> go to the next tab with the right arrow and press enter. The navigation did not run.

### Expected behavior

The keyboard navigation should work as standard tabs: the navigation should start when the enter button is pressed on the new tab.

**Comment by Retch** [comment]
Should be the same like [this](https://github.com/primefaces/primeng/issues/17774)
